src/graphics/constants.js

```diff
@@ -984,6 +984,13 @@ export const TEXTURETYPE_RGBM = 'rgbm';
  */
 export const TEXTURETYPE_RGBE = 'rgbe';
 
+/**
+ * Texture stores high dynamic range data in RGBP encoding.
```

Comment:
Wondering if some detail on the format is helpful here. Same goes for RGBM and RGBE I guess.

Reply:
Agree, will add something to the engine in a future PR.